test/lib/getSrcForJoin.test.js

```diff
@@ -22,3 +26,59 @@ describe('getSrcForJoin', () => {
         assert.deepEqual(getSrcForJoin(WORKFLOW, { jobName: 'bar' }), []);
     });
 });
+
+describe('isPR', () => {
+    const isPR = rewireGetSrcForJoin.__get__('isPR');
+
+    it('sholud return true if job name has PR prefix', () => {
+        assert.isTrue(isPR('PR-1:testJobName'));
+    });
+
+    it('sholud return false if job name does not have PR prefix', () => {
+        assert.isFalse(isPR('testJobName'));
+    });
+});
+
+describe('findJobs', () => {
+    it('should find jobs when search job is joined job', () => {
+        const findJobs = rewireGetSrcForJoin.__get__('findJobs');
+        const destJobName = 'foo';
+
+        assert.deepEqual(findJobs(WORKFLOW, destJobName),
+            new Set([{ name: 'main', id: 1 }, { name: 'other_main', id: 2 }]));
+    });
+
+    it('should not find jobs when search job is not joined job', () => {
+        const findJobs = rewireGetSrcForJoin.__get__('findJobs');
```

Comment:
Can this `findJobs` share with these test cases same as `isPR` test?

Reply:
fixed